README.md

```diff
@@ -209,6 +216,7 @@ Model,DemoUAV
 Serial number,AJ34NF12
 Firmware version,1.75.24a_rev2
 Propeller PPK version,1.0
+Camera Serial Number,3742BE02
```

Comment:
Do we need this in both the csv & exif? Can we pick one or the other?

Reply:
This was added in case the camera doesn't include the serial number and the drone manufacturer doesn't have control over the EXIF data; They would be able to add it in here